app/src/main/res/values/strings.xml

```diff
@@ -56,9 +56,11 @@
     <string name="position_name">Position name</string>
     <string name="branch">Branch</string>
     <string name="group">Group</string>
-    <string name="off_have_salary">Off have salary</string>
+    <string name="off_have_salary_company_pay">Off have salary - Company Pay</string>
+    <string name="off_have_salary_insurance_coverage">Off have salary - Insurance Coverage</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.off_have_salary_insurance_coverage` appears to be unused
 Unused resources make applications larger and slow down builds.